ansible/roles/elastic-beats/tasks/main.yml

```diff
@@ -0,0 +1,140 @@
+---
+
+- name: copy filebeat ILM policy
+  template:
+    src: 'ilm_policy.json.j2'
+    dest: '/etc/filebeat/ilm_policy.json'
+    mode: 0644
+    owner: root
+    group: root
+
+- name: set up filebeat log monitoring
+  include_role:
+    name: elastic.beats
+  vars:
+    beat: filebeat
+    beat_conf:
+      setup:
+        template:
+          name: logs-drive-abci
+          pattern: logs-drive-abci-*
+          append_fields:
+            - name: json.stateRepository.params
+              type: object
+              dynamic: runtime
+            - name: json.stateRepository.response
+              type: object
+              dynamic: runtime
+            - name: json.abciRequest
+              type: object
+              dynamic: runtime
+            - name: json.errors
+              type: nested
+              dynamic: runtime
+            - name: json.height
+              type: long
+            - name: json.err
+              type: object
+              dyanmic: runtime
+          settings:
+            index:
+              sort:
+                field:
+                  - json.height
+                  - json.abciMethod
+                  - json.level
+                order:
+                  - asc
+                  - asc
+                  - asc
+        ilm:
+          policy_file: /etc/filebeat/ilm_policy.json
+      filebeat:
+        inputs:
+          - type: log
+            enabled: true
+            json.message_key: msg
+            json.keys_under_root: false
+            exclude_files: ['\.gz$']
+            paths:
+              - "{{ mn_evo_services_path }}/logs/*.log*"
+      processors:
```

Comment:
We can move processors under `input`: https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-log.html#_processors_10